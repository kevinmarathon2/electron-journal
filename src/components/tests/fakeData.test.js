const fakeData = require("../fakeData.js");

test("fake data is not empty", () => {
  expect(fakeData).toMatchSnapshot();
});

test("Fake data is an object", () => {
  expect(typeof fakeData).toBe("object");
});
