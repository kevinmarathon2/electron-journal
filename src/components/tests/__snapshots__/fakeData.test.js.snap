// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fake data is not empty 1`] = `
Array [
  Object {
    "data": Array [
      Object {
        "element": "div",
        "text": "li item first on the hournal",
      },
      Object {
        "element": "div",
        "text": "line two",
      },
    ],
  },
  Object {
    "data": Array [
      Object {
        "element": "div",
        "text": "li item first on the hournal",
      },
      Object {
        "element": "div",
        "text": "line two",
      },
    ],
  },
]
`;
